src/components/switch/_switch.scss

```diff
@@ -0,0 +1,106 @@
+/* ===============Switch================== */
+.cmp-adaptiveform-switch {
+  color: $dark-gray;
+  font-size: $font-md;
+  margin: $field-margin;
+}
+.cmp-adaptiveform-switch__widget-container {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+  margin-bottom: 5px;
+}
+
+.cmp-adaptiveform-switch__container {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.cmp-adaptiveform-switch__widget {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.cmp-adaptiveform-switch__widget-slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 34px;
+}
+
+.cmp-adaptiveform-switch__widget-slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .cmp-adaptiveform-switch__widget-slider {
+  background-color: #2196F3;
+}
+
+input:focus + .cmp-adaptiveform-switch__widget-slider {
+  box-shadow: 0 0 1px #2196F3;
+}
+
+input:checked + .cmp-adaptiveform-switch__widget-slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(26px);
+}
+
+.cmp-adaptiveform-switch__widget-slider:before {
+  border-radius: 50%;
+}
+.cmp-adaptiveform-switch__longdescription {
+  color: $light-gray;
+  background-color: $very-light-gray;
+  font-size: $font-sm;
+  margin-top: $error-top-margin;
+  margin-bottom: 5px;
+  padding: 10px;
+  p{
+    margin: 0;
+    padding: 0;
+  }
+}
+.cmp-adaptiveform-switch__questionmark{
+  display:inline-block;
+  width:1rem;
+  height:1rem;
+  border-radius: 9px;
+  background: url(./resources/images/question.svg) center center / cover no-repeat,#969696;
+  cursor : pointer;
+  // below properties for backward compatibility
+  position: absolute;
+  right: 20px;
```

Comment:
@egmox Have you validated with RTL?

Reply:
@iamsudhanshu RTL is validated. There is a class for rtl at line #89